api/graphql/schema.graphql

```diff
@@ -244,6 +244,7 @@ input CreateUserInput {
 When updating a User, all fields are optional, and email cannot be changed.
 """
 input UpdateUserInput {
+    id: ID
```

Comment:
Does the id need to be added to the UpdateUserInput? Since the id isn't altered during the update process.

Reply:
Hehe, yeah, this was the source of my question to #devspam on the 8th.  Without it the API complains that the primary key is missing for the update.  You would think that the model relationship would tell it which user is being updated but without it explicitly being specified it will fail.  It's a bad smell but I don't know of a way around it.